index.html

```diff
@@ -0,0 +1,130 @@
+            <i class="fa-solid fa-xmark"></i>
+          </ul>
+          <Span>Special Seat</Span>
+        </nav>
+        </div>
+    </header>
+    <div class="begin">
+        <div class="introduction">
+            <div class="intro-heading">
+                <p>"Hello HennaHub Globe!"</p>
+                <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+            </div>
+            <p class="intro-text">A joyful celebration of creativity and mehndi design for the first apprentice graduation og HennaHubUK academy</p>
+            <h2>2023-12-20(WED) ~ 21(THU)</h2>
+            <span>@ 1312 Gallowgate Glasgow G31 4DR UK</span>
+        </div>
+    </div>
+    <div class="main-program">
+        <div>
+        <h2>Main Program</h2>
+        <hr>
+    </div>
+        <div class="program">
+            <section class="program-list">
+                <div class="card-display">
+                <div class="program-image">
+                    <img src="./images/exhibitionicon (2).svg" alt="lecture icon">
+                    <p class="program-name">Lecture</p>
+                    <p class="program-text">
+                        Listen to honourable speakers from various locations about the mission of hennahubuh mehndi creations 
+                    </p> 
+                </div>
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src="./images/networkicon (3).svg" alt="exhibition icon">
+                    <p class="program-name">Exhibition</p>
+                    <p class="program-text">
+                        Showcasing models with various mehndi designs and apreciating other mehndi creativities on materials 
+                    </p> </div>
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src="./images/networkicon (4).svg" alt="Workshop Icon">
+                    <p class="program-name">Workshop</p>
+                    <p class="program-text">
+                        Workshops on the various creativity done by HennaHubUk academy
+                    </p>
+                </div> 
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src= "./images/lectureicon (1).svg" alt="Networking icon">
+                    <p class="program-name">Networking</p>
+                    <p class="program-text">
+                        Get connected with fan around the world and make new friends even after the summit
+                    </p>
+                </div> 
+                </div>  
+            </section>
+            <button> JOIN HENNAHUB SUMMIT 2023</button>
+        </div>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </div>
```

Comment:
- [ ] Nice work implementing the programs section :clap: However, the link to "see the whole program" it is not showing on the desktop version and instead of 5 programs you have 4 programs. Kindly make sure you stick to the provided design as much as possible to follow the project requirement.
### Your design :point_down: 
![Screenshot from 2023-07-06 10-38-25](https://github.com/Zuwairanajma/ModuleOne-Capstone-Project/assets/91142432/1eb37b7e-17ca-4b6e-a06e-49410922cfe1)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-07-06 10-39-08](https://github.com/Zuwairanajma/ModuleOne-Capstone-Project/assets/91142432/e2db273d-606e-43b9-9584-f9cd5e09f174)